internal/document/frontmatter.go

```diff
@@ -75,8 +75,6 @@ func toJSONStr(f *Frontmatter, source []byte, requireIdentity bool) (string, err
 	if requireIdentity {
 		f.ensureID()
 		m["runme"] = f.Runme
-	} else {
```

Comment:
Here we are avoiding removing existent frontmatter metadata, but what happens if the **`runme`** metadata is wrong or invalid?

Reply:
Like an invalid ID (ulid format) or version number (not following the vx.x convention) @pastuxso? You have some examples of invalidity?

Reply:
@sourishkrout i.e. any hardcoded string in the metadata.

Reply:
I think we should be super conservative with `IDs`. Unless we really can't work with it, we shouldn't reset it. The version we can just overwrite since it does not constitute "identity". Does that help @pastuxso?

Reply:
@sourishkrout Yes, that helps, I think we can tackle it in an upcoming task.